### PR TITLE
[OT185-62] Send email when new Contact is created

### PR DESCRIPTION
**Pull request by locknet11**
## Summary
This PR modifies: 
 - SendGridEmailService
 - EmailTemplateValues
 - EmailService
 - application.properties

### Type of change
- [x] New feature
- [ ] New Tests
- [ ] Bug fix
- [ ] Refactor

### Checklist

- [x] Traceability between this change and Jira.
- [x] ```mvn clean install``` was run and all tests completed successfully.
- [x] There are no unused variables and/or imports in the modified classes.
- [x] There are no imports in the modified classes with the wildcard character. Ex: ```com.somepackage.*```.
- [x] Slf4j was used for the application log instead ```System.out```.
- [x] Public methods are documented with ```javadoc```.

### CODE REVIEW

File: src/main/java/com/alkemy/ong/ports/output/email/SendGridEmailService.java [hunk 1]
```diff
@@ -58,27 +62,35 @@ protected void init() {
...
-		personalization.addDynamicTemplateData(ORGANIZATION_FACEBOOK, organization.getFacebookUrl());
 		personalization.addDynamicTemplateData(SUBJECT, WELCOME_SUBJECT);
+		personalization.addDynamicTemplateData(WELCOME_TEXT, organization.getWelcomeText());
 		personalization.addTo(emailTo);
-
 		mail.addPersonalization(personalization);
+		sendMail(mail);
+	}
 
+	@Override
```
Comment (manerajona) [inline src/main/java/com/alkemy/ong/ports/output/email/SendGridEmailService.java:78]: Podríamos hacerlo async también

File: src/main/java/com/alkemy/ong/domain/usecase/impl/ContactServiceImpl.java [hunk 2]
```diff
@@ -22,7 +33,11 @@ public List<Contact> getList() {
 
 	@Override
 	@Transactional
-	public Long createEntity(Contact contact){
+	public Long createEntity(Contact contact) {
+		Optional<Organization> organization = organizationJpaRepository.findById(ORGANIZATION_ID);
+		if(organization.isPresent()) {
```
Comment (manerajona) [inline src/main/java/com/alkemy/ong/domain/usecase/impl/ContactServiceImpl.java]: Podríamos refactorizarlo y hacer:
```java
Optional<Organization> op = organizationJpaRepository.findById(ORGANIZATION_ID);
op.ifPresent(organization -> emailService.sendContactedSuccessfully(contact, organization));
```

**Review by manerajona — CHANGES_REQUESTED** [review]

File: src/main/java/com/alkemy/ong/ports/output/email/SendGridEmailService.java (hunk 1 above)
Comment (locknet11) [reply src/main/java/com/alkemy/ong/ports/output/email/SendGridEmailService.java:78]: Listo

File: src/main/java/com/alkemy/ong/domain/usecase/impl/ContactServiceImpl.java (hunk 2 above)
Comment (locknet11) [reply src/main/java/com/alkemy/ong/domain/usecase/impl/ContactServiceImpl.java]: Listo

**Pull request merged**